src/api/userManagementApi.ts

```
@@ -216,113 +221,185 @@ export const createUser = async (userData: User): Promise<User> => {
     case "Teacher":
-      formattedData = {
-        ...formattedData,
-        teacherData: {
-          teacherGrade: userData.grade || '',
-          teacherClass: userData.class || '',
-          subject: userData.subject || '',
-          medium: userData.medium || '',
-          staffNo: userData.staffNo || '',
-        }
-      };
+      const teacherData: Record<string, any> = {};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)